tests/test_crud.py

```diff
@@ -17,6 +17,17 @@
     return u
 
 
+async def test_create_from_instance(engine):
+    nickname = 'test_create_from_instance_{}'.format(random.random())
+    u = User(nickname='will-be-replaced', type=UserType.USER)
+    u.nickname = nickname
+    await u.create(bind=engine, timeout=10)
+    assert u.id is not None
+    assert u.nickname == nickname
+    assert u.type == UserType.USER
```

Comment:
After these checks, it is better to load the same user from database again and make a double check to avoid potential cache issues. Also please add `age` back after JSON property gets fixed.